controller/source/gdpr_metadata.hpp

```diff
@@ -4,11 +4,28 @@
 #include <algorithm>
 #include <vector>
 #include <unordered_map>
+#include <chrono>
+#include <iomanip>
+#include <sstream>
 
 namespace controller {
 
 constexpr int num_users = 64;
 constexpr int num_purposes = 64;
+constexpr int metadata_prefix_fields = 8;
+
+enum metadata_fields {
+  usr,
+  encr,
+  pur,
+  obj,
+  org,
+  exp,
+  shr,
+  log,
```

Comment:
I can match each metadata field to its corresponding short name (user_key -> usr, encryption -> encr, etc). But I could not match the `monitor` field to a short name. Should it be `log`? If so, shall we consider renaming it to `mon` maybe to keep consistency?

Reply:
The `monitor` field was used as this is how it's named in the query part but it refers to logging.
I will consider changing it when finishing the filtering part.
Although I think it would be better to keep it `log` or change it to `monitor` cause `mon` can be misleading/strage.
Your understanding is correct though.